vsphere/tests/test_vsphere.py

```diff
@@ -428,14 +424,32 @@ def test__collect_metrics_async_compatibility(vsphere, instance):
 
     vsphere.in_compatibility_mode.return_value = True
     vsphere._collect_metrics_async(instance, [])
-    vsphere.log.debug.assert_called_with("Skipping unknown `unknown` metric.")
+    vsphere.log.debug.assert_called_with('Skipping unknown `%s` metric.', 'unknown')
 
     vsphere.log.reset_mock()
     vsphere.in_compatibility_mode.return_value = False
     vsphere._collect_metrics_async(instance, [])
     vsphere.log.debug.assert_not_called()
 
 
+def test__collect_metrics_async_hostname(vsphere, instance, aggregator):
```

Comment:
why the two __?

Reply:
Yeah it's a bit of weird pattern. Because the method is private `_collect`, with the `test_` prefix it makes two.